### DISCUSSION TITLE
Disable ScrollViewer Scroll To Control

### DISCUSSION THREAD

**Discussion by AdamA-SVK** [AvaloniaUI/Avalonia#16613] [Q&A] [answered]
I have ScrollViewer with many controls. When I interact with any controll at the bottom and then I use some from the top, ScrollViewer scrolls to bottom to the previously used control. It happens only when I click on MenuItem from Menu or when I switch between windows. How to stop this behavior?
It is the same case like in WinForms [Panel with AutoScroll on](https://stackoverflow.com/questions/419774/how-can-you-stop-a-winforms-panel-from-scrolling/912610)

Video example:
https://github.com/user-attachments/assets/830dcf6e-9890-4a3f-b117-b0667ad96cde

**Comment by grokys** [comment, marked as answer]
Hi @AdamA-SVK ,

Try setting `ScrollViewer.BringIntoViewOnFocusChange="False"` to disable this behavior. For more information see https://github.com/AvaloniaUI/Avalonia/pull/11442

**Reply by AdamA-SVK** [reply]
This is just great! Thank you
